cli/args/flags.rs

```diff
@@ -237,10 +237,16 @@ pub struct InstallFlagsGlobal {
 
 #[derive(Clone, Debug, Eq, PartialEq)]
 pub enum InstallKind {
-  Local(Option<AddFlags>),
+  Local(InstallFlagsLocal),
   Global(InstallFlagsGlobal),
 }
 
+#[derive(Clone, Debug, Eq, PartialEq)]
+pub struct InstallFlagsLocal {
+  pub add_flags: Option<AddFlags>,
+  pub entrypoints: Option<Vec<String>>,
```

Comment:
Isn't `None` the same as `vec![]`?

Reply:
Yeah pretty much. I just find `Option` makes it clearer the flag is optional. Regardless, I refactored this to an enum since it's clearer.